libsweep/README.md

```diff
@@ -21,7 +21,7 @@ For windows users, open a command prompt with administrative access:
 
     mkdir build
     cd build
-    cmake .. -DCMAKE_BUILD_TYPE=Release
+    cmake ..
```

Comment:
Hm but on other system's a Release config is not the default.

We might not really care, but for Linux and macOS this decides what kind of optimization flags CMake passes to gcc.